### PR TITLE
Update 02-Ray-Serve-Example.ipynb

### PR DESCRIPTION
**Pull request by anthony0727**
import sklearn to run sklearn.utils.shuffle

### CODE REVIEW

**Comment by mGalarnyk** [comment]
Merged! Thanks for your contribution!
